test/topology/util.py

```diff
@@ -234,6 +235,48 @@ async def check_system_topology_and_cdc_generations_v3_consistency(manager: Mana
         # Check that the contents fetched from the current host are the same as for other nodes
         assert topo_results[0] == topo_res
 
+async def start_writes_to_cdc_table(cql: Session, concurrency: int = 3):
```

Comment:
I'm not sure if this function should be defined in `util.py`. It does a lot, it's barely configurable and I doubt it will be reused. I would define it in the test file.

Reply:
It will be used also in the topology recovery test.